bot/cogs/git.py

```diff
@@ -131,6 +131,24 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    @git.command(help'Prints info on the git pull command')
+    async def pull(self, ctx):
+        msg = ManMessage(
+            name='git-pull'
+            synopsis='git pull git pull [options] [<repository> [<refspec>...]]',
```

Comment:
Remove double `git pull`. Also, please specify all options mentioned here, like repository and refspec, through the options argument below.